fhirstore/search_engine.py

```diff
@@ -0,0 +1,82 @@
+from yarl import URL
+
+from fhirpath.connectors.factory.es import ElasticsearchConnection
+from fhirpath.engine.es import ElasticsearchEngine as BaseEngine
+from fhirpath.engine import dialect_factory
+from fhirpath.enums import FHIR_VERSION
+from fhirpath_helpers.elasticsearch.mapping import generate_mappings
+
+
+class ElasticSearchEngine(BaseEngine):
+    def __init__(self, es_client, fhir_release=FHIR_VERSION.R4):
+        super().__init__(
+            fhir_release, lambda x: ElasticsearchConnection(es_client), dialect_factory
+        )
+        self.mappings = generate_mappings(FHIR_VERSION.R4.name)
+
+    def calculate_field_index_name(self, resource_type):
+        return resource_type
+
+    def current_url(self):
+        """
+        complete url from current request
+        return yarl.URL"""
+        return URL("https://dev.arkhn.com")
```

Comment:
You left a hard-coded url here

Reply:
done

Reply:
Is it really a FHIR_API?
Asking because it was only "https://dev.arkhn.com" before

Reply:
not sure it's really that, I think it has to do with pagination. We'll deal with this later if that's ok.